src/js/pensions/config/form.js

```diff
@@ -199,10 +199,28 @@ const formConfig = {
               'ui:widget': 'yesNo'
             },
             severancePay: {
-              'ui:title': 'Pay type',
-              'ui:widget': 'RadioWidget',
+              'ui:order': [
+                'type',
+                'amount'
+              ],
               'ui:options': {
                 expandUnder: 'view:receivedSeverancePay'
+              },
+              amount: {
+                'ui:title': 'Amount'
+              },
+              type: {
+                'ui:title': 'Pay Type',
+                'ui:widget': 'radio',
+                'ui:options': {
```

Comment:
Thanks @jbalboni, I thought so too initially, but removing these causes a `enumOptions.map is not a function` error.